### DISCUSSION TITLE
How to set the text center of tabviewitem

### DISCUSSION THREAD

**Discussion by bbhxwl** [xamarin/XamarinCommunityToolkit#1089] [Q&A]
How to set the text center of tabviewitem

**Comment by Cfun1** [comment]
Hi, it seems more of a question, if so you please close the issue and open a question in the [discussion section](https://github.com/xamarin/XamarinCommunityToolkit/discussions) instead. If it is a feature request could you please check if it isn't a duplicate of https://github.com/xamarin/XamarinCommunityToolkit/issues/776 ?

**Comment by pictos** [comment]
@jsuarezruiz any thoughts?
